README.md

```diff
@@ -118,6 +118,27 @@ Unpause Registration:
 docker run --rm --network host --env-file .env -v ./.nodes:/root/.nodes wavs-middleware unpause
 ```
 
+Delegation to Operator:
+
+```bash
+# Generate a new private key for the staker (needs ETH for transactions)
+STAKER_KEY=$(cast wallet new --json | jq -r '.[0].private_key')
+STAKER_ADDRESS=$(cast wallet addr --private-key "$STAKER_KEY")
+echo "Staker address: $STAKER_ADDRESS"
+
+export DELEGATION_MANAGER_ADDRESS=0xA44151489861Fe9e3055d95adC98FbD462B948e7
+
+docker run --rm --network host  --env-file .env -v ./.nodes:/root/.nodes \
+   -e STAKER_KEY=${STAKER_KEY} \
+   -e OPERATOR_ADDRESS=${OPERATOR_ADDRESS} \
+   -e WAVS_SERVICE_MANAGER_ADDRESS=${WAVS_SERVICE_MANAGER_ADDRESS} \
+   -e DELEGATION_MANAGER_ADDRESS=${DELEGATION_MANAGER_ADDRESS} \
+   -e DELEGATION_APPROVER_PRIVATE_KEY=${OPERATOR_KEY} \
```

Comment:
This should not require operator private key.

We want to simulate some random staker putting in some sETH on a given operator.

It should need to know the wavs service manager and the operator address to delegate to, not a private key.

I think DELEGATION_MANAGER_ADDRESS can be queried from the WAVS_SERVICE_MANAGER, which owuld make this easier to configure